src/utils/ug-utils.js

```diff
@@ -3,34 +3,28 @@ function combineAndSortBodyFields (content) {
 	let stickyContent = [];
```

Comment:
The combineAndSortBodyFields function doesn't appear to be used anywhere. I can't remember the history of it, but if we're not using it, we should get rid of the whole function.

Reply:
Done